src/App.tsx

```diff
@@ -2,108 +2,47 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
+
+function getUserInfo(userID: number): User {
+  const foundUser = usersFromServer.find(user => user.id === userID);
+
+  return foundUser || {
+    id: userID,
```

Comment:
Instead of hardcoding the user data, return a default user object with a message indicating that the user was not found.